test/typescript/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+// import { useSelector } from 'react-redux';
+
+const equal = require('../..');
+
+const testArr = [
+  { text: 'green', id: '23' },
+  { text: 'sunshine', id: '1' },
+  { text: 'mountain', id: '11' },
+  { text: 'air', id: '8' },
+  { text: 'plants', id: '9' },
+] as ITodo[];
```

Comment:
we should be able to type this without a cast. usually `as` is a code smell in TS.

try `const testArr : ITodo[] = [` instead